typings/connection.d.ts

```diff
@@ -459,11 +459,11 @@ export interface MessageProperties {
   /**
    * @property {number} [absolute_expiry_time] The time when this message is considered expired.
    */
-  absolute_expiry_time?: number;
+  absolute_expiry_time?: number | Date;
```

Comment:
I wonder if it would be better to change the type, rather than allow either. That way what you put in is always what you get out. What do you think?

Reply:
Makes total sense to me. The fact that it's marked as a number would end up being a lie - I wasn't sure if you wanted to potentially gate this behavior with a configuration option (in which case it could be either).